wasm/src/transaction/transaction.rs

```diff
@@ -57,25 +57,19 @@ impl Transaction {
     pub fn transaction_id(&self) -> String {
         self.transaction.transaction_id().to_string()
     }
-
```

Comment:
Do we still expose all the properties of transaction as a string here? We might be missing some since this hasn't been updated in a while.

Reply:
This is a good idea. Implementing in a separate pr.